lib/galaxy/datatypes/tabular.py

```diff
@@ -1161,3 +1161,47 @@ def get_chunk(self, trans, dataset, chunk):
         ck_data_body = re.sub('[ ]+', '\t', ck_data_body)
 
         return dumps({'ck_data': util.unicodify(ck_data_header + "\n" + ck_data_body), 'ck_index': ck_index + 1})
+
+
+@dataproviders.decorators.has_dataproviders
+@build_sniff_from_prefix
+class Gro(Tabular):
+    file_ext = 'gro'
+
+    def __init__(self, **kwd):
+        """Initialize gro datatype"""
+        super(Gro, self).__init__(**kwd)
+
+
+    def display_peek(self, dataset):
+        """Returns formated html of peek"""
+        return self.make_html_table(dataset)
+
+    def sniff_prefix(self, file_prefix):
```

Comment:
This sniffer seems to me not to strict enough. If we can not reliably sniff a GRO files we should not do it and just subclass the GRO datatype from tabular in the datatypes_conf.xml file. So this python class is not needed.